Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md

```diff
@@ -0,0 +1,38 @@
+
+# Audio/Video in Google Chrome
+
+
+
+ _**Applies to:** Skype for Business 2015_
+
+At the moment we only support one incoming video channel in Google Chrome. 
+To still render video of multiple participants in a conversation we have introduced the **Active Speaker** API.
+
+Through the API the video of the currently speaking participant is sent over the one available channel.
+
+## Active Speaker API
+
+### How fo I know when to use the Active Speaker API?
+```js
+    var conversation = application.conversationsManager.createConversation();
+    if (conversation.videoService.videoMode() === 'ActiveSpeaker') {
+        // use the Active Speaker API
+    }
+```
+
+### How do I render the Active Speaker Channel?
+```js
+    // set up listener to detect if someone is actively speaking
+    conversation.videoService.activeSpeaker.participant.changed(function (participant) {
```

Comment:
To my knowledge this listener can be added even before. It doesn't matter if the participant is UNDEFINED to begin with.